src/OVAL/probes/unix/file.c

```diff
@@ -80,6 +81,8 @@
 # endif
 #elif defined(_WIN32)
 # define OS_WINDOWS
+#elif defined(_AIX)
+# define OS_AIX
```

Comment:
this define is never used, what is it for?

Reply:
Yes I know and this is intended for at least the 2 following reasons:
I had to do it that way else the pre processor would fall there:
# error "Sorry, your OS isn't supported."
Moreover, please leave this #define OS_AIX because I have to add in a future patch some code specific to AIX to implement acl_trival function (or equivalent) for the has_extended_acl function.

Reply:
Aha, OS_WINDOWS is never used. I though it is used in master branch, but it isn't used there as well.